### DISCUSSION TITLE
90AE471C-0406-4821-A793-D83893D2D0B2

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#79] [General]
# Kepler-48

Kepler-48, also known as KOI-148, is a K-type main sequence star about 1,009 light-years away in the constellation Cygnus. Between 2012 and 2014, four planets (three superterrans and one superjovian) are discovered around this star

https://celestia.mobi/addon?item=90AE471C-0406-4821-A793-D83893D2D0B2&lang=en

**Comment by exocritic5457** [comment]
This should have a remake, there isn't really a better excuse than another planet being found imho

**Reply by Karistus** [reply]
It has been done

**Reply by exocritic5457** [reply]
I appreciate the remakes, they actually aren't hard on my eyes. Excited for 80

**Comment by kmjordan122313-lgtm** [comment]
ooh, a nice addon fr!
